tests/multi-platform/multi-platform.go

```diff
@@ -0,0 +1,236 @@
+package multiplatform
+
+import (
+	"context"
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/devfile/library/v2/pkg/util"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appservice "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	buildservice "github.com/redhat-appstudio/build-service/api/v1alpha1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+)
+
+const (
+	Ec2ArmTag           = "multi-platform-e2e-arm64"
+	HostConfig          = "host-config"
+	ControllerNamespace = "multi-platform-controller"
+	SecretName          = "awskeys"
+)
+
+var (
+	testProjectGitUrl   = utils.GetEnv("MULTI_PLATFORM_TEST_REPO_URL", "https://github.com/devfile-samples/devfile-sample-go-basic")
+	testProjectRevision = utils.GetEnv("MULTI_PLATFORM_TEST_REPO_REVISION", "c713067b0e65fb3de50d1f7c457eb51c2ab0dbb0")
+)
+
+var _ = framework.JVMBuildSuiteDescribe("Multi Platform Controller E2E tests", Label("multi-platform-controller"), func() {
```

Comment:
Would it be good to use another wrapper function here like `MultiPlatformBuildSuiteDescribe`